src/plugin.py

```diff
@@ -191,8 +195,40 @@ def _force_utc(self):
             logging.exception("Can not parse backend response")
             raise UnknownBackendResponse()
 
+
+        self._own_games = games
+        
+        game_ids = list(map(lambda x: x.game_id, owned_games))
+        other_games = await self.get_steam_sharing_games(game_ids)
+        for i in other_games:
+            owned_games.append(i)
+
         return owned_games
 
+    async def get_steam_sharing_games(self,owngames: List[str]) -> List[Game]:
+        profiles = list(filter(lambda x: "!" in x.user_name, self._own_friends))
```

Comment:
Add explaination why are we serching for "!"
Can a user create a username with "!" himself ?

Reply:
You can change the nickname of your friends in the steam client and include some pattern which I now have set as "!". I can't find a way to automatically get which friends have shared games with you so this is a workaround.

Reply:
Ah I see what you mean. I wish there was plugin configuration on galaxy integration plugins so we can select and store the list of friends there.
What we can do is check for both ending with "*" (which means nickname) and "!"